src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -975,7 +975,135 @@ function setGaugeStyleSheet(gaugeName, css, cssback, csstext)
   setLabelStyleSheet(gaugeName .. "_text", csstext or "")
 end
 
+-- https://wiki.mudlet.org/w/Manual:Lua_Functions#getHTMLspan
+-- used by xEcho for creating formatting span tags for labels
+-- fmt is a table of format options as returned by getTextFormat
+function getHTMLspan(fmt)
+  -- next two lines effectively invert the colors if fmt.reverse is true
+  local fore = fmt.foreground
+  local back = fmt.background
+  if fmt.reverse then
```

Comment:
Could you localise `type` and `string.format` here, as they're used a couple of times?